frontend/src/libs/enums/components/article-form/article-form-submit-type.enum.ts

```diff
@@ -0,0 +1,6 @@
+const SubmitType = {
```

Comment:
And name enum the same way + not sure if it should be in common libs/enums - as it is related only to articles form
```suggestion
const ArticleSubmitType = {
```